dimensiondata/src/test/java/org/jclouds/dimensiondata/cloudcontrol/internal/BaseDimensionDataCloudControlApiLiveTest.java

```diff
@@ -89,8 +89,17 @@ public void setUp() {
       datacenters = getZones();
    }
 
+   //   private Set<String> getZones() {
+   //      return ctx.utils().injector().getInstance(ZoneIdsSupplier.class).get();
+   //   }
+
+   // TODO this leads to a warning - WARNING: failed to find key for value https://api-na.dimensiondata.com in {au=https://api-au.dimensiondata.com}; choosing first: au
```

Comment:
Would like some help with this TODO if possible.

Reply:
Can you send an email to the dev@ mailing list with the details of what you are trying to achieve, the current config, and the failure? Just to have the whole context outside this PR and we can help properly.

Reply:
Thanks @nacx I will send that mail.